ThermalTalk.QueryStatus/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace ThermalTalk.QueryStatus
+{
+    using System;
+
+    /// <summary>
+    /// Simple console app to query the printer status
+    /// </summary>
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            const string reliancePort = "COM12";
```

Comment:
Can you make this grab the port from args? e.g. 

`var portName = args.FirstOrDefault()` etc. Then show an error/usage message if they don't provide a port name?